### PR TITLE
Set up for Documentation

### PR DESCRIPTION
**Pull request by LuisOlivaresJ**
1. New **docs** folder to found **Documentation** structure
* ``docs/source/index.rst`` is the root document (root of the “table of contents tree” or **toctree**)
* ``README.rst`` continues to be the welcome page, but using a new format ([reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html))
* ``docs/source/installation.rst`` for installation instructions
* The main module overview is imported from the corresponding ``Docstring`` (E. g. from src/omg_dosimetry/calibration.py docstring) 

2. To do: import our documentation
2.1 [Create an account](https://readthedocs.org/accounts/signup/) on Read The Docs
2.2 Sign up or log in at Read the Docs with your GitHub account
2.3  Import our documentation repository ([here is a guide](https://docs.readthedocs.io/en/stable/intro/import-guide.html#importing-your-documentation)). Just for meantime, I suguest to use the "Documentation" branch instead the base "OMG_master"
2.4 On Administration, include me as a Maintainer

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+OMG Dosimetry, as a scientific package, has fairly standard scientific dependencies (>= means at least that version or newer).
+Installing the package via ``pip`` will install these for you:
+
+.. literalinclude:: ../../requirements_pip.txt
+
+I'm new to Python
+=================
+
+That's okay! If you're not a programmer at all you'll have a few things to do to get up. Using pylinac requires not just the base language Python, but a few dependencies as well.
```
Comment (jfcabana) [inline docs/source/installation.rst]: Should change references to pylinac.

File: docs/source/installation.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Since most physicists don't program, or if they do it's in MATLAB, this section will help jumpstart your use of not
+just pylinac but Python in general and all its wonderful goodness! Getting started with Python takes some work to
```
Comment (jfcabana) [inline docs/source/installation.rst]: same here

**Comment by jfcabana** [comment]
I followed the steps to setup the project on Read the Docs. The compilation failed. Seems to be related to the `requirements.txt `file?

image

**Comment by LuisOlivaresJ** [comment]
Fixed. It seems that Read The Docs was trying to automatically use "requirements.txt" instead "requierements_read_the_docs.txt". I just renamed the first one and it worked. I am considering to delete it.

Have you see the [output page](https://omg-dosimetry.readthedocs.io/en/latest/index.html)? It is beautiful :blush:

**Comment by jfcabana** [comment]
Yep, looking good! 

I thought the docstrings from the three modules would appear in the docs, but that does not seem to be the case?

**Comment by LuisOlivaresJ** [comment]
Do you mean the docs folder? 
For example, the content in [calibration.rst](https://github.com/jfcabana/omg_dosimetry/blob/Documentation/docs/source/calibration.rst) file, is going to be the information to be show for the Calibration module on the web page. 
This is how it works, with the addition of two lines of code (see #7), 

Captura

All the docstring from LUT's class (from PyPi omg_dosimetry package) is going to be show on the [web page](https://omg-dosimetry.readthedocs.io/en/latest/calibration.html#omg_dosimetry.calibration.LUT).

**Comment by jfcabana** [comment]
Yes, that is it! There are several classes and functions that I think would be a good idea to include in the documentation.

**Review by jfcabana — APPROVED** [review]
I will need to update some of that documentation, but we can merge this as is for now to get started.

**Comment by LuisOlivaresJ** [comment]
Perfect. I'll work on installation.rst to take acount about the spyder requeriment.

**Pull request merged**